src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/CheckFunding.cshtml

```diff
@@ -1,7 +1,14 @@
-﻿
+﻿@using SFA.DAS.EmployerAccounts.Web.FeatureToggles
 @model SFA.DAS.EmployerAccounts.Web.ViewModels.AccountDashboardViewModel
 
 <h3 class="das-panel__heading">Check if government funding is available for apprenticeship training </h3>
 <p>Use government funding to pay for part or all of the cost of apprenticeship training.</p>
 
-<p><a href="#" class="das-panel__link">Check and secure funding now</a> </p>
+@if (Features.EmulatedFundingJourney.Enabled)
```

Comment:
Can this go into a URL helper (HtmlHelperExtensions extension methods).  Then the view doesn't know about the feature toggle.

Reply:
Done

Reply:
Prroperly done now